### PR TITLE
Be able to skip not readable entries in a Zip Stream

### PR DESCRIPTION
**Pull request by carlosthe19916**

### CODE REVIEW

File: utils/src/main/java/org/jboss/windup/util/ZipUtil.java [hunk 1]
```diff
@@ -186,4 +186,18 @@ public static List<String> scanZipFile(String parentPath, InputStream is, boolea
         }
     }
 
+    /**
+     * Return the next Entry of a ZipInputStream. If the nextEntry can not be read then kip it and continue with the next one
+     * @param zis
+     * @return next zip entry
+     * @throws IOException
+     */
+    public static ZipEntry getNextEntry(ZipInputStream zis) throws IOException {
```
Comment (jmle) [inline utils/src/main/java/org/jboss/windup/util/ZipUtil.java:195]: Does this method have to be public?

File: utils/src/main/java/org/jboss/windup/util/ZipUtil.java [hunk 2]
```diff
@@ -186,4 +186,18 @@ public static List<String> scanZipFile(String parentPath, InputStream is, boolea
         }
     }
 
+    /**
+     * Return the next Entry of a ZipInputStream. If the nextEntry can not be read then kip it and continue with the next one
```
Comment (jmle) [inline utils/src/main/java/org/jboss/windup/util/ZipUtil.java:190]: Typo in "kip"

**Review by jmle — CHANGES_REQUESTED** [review]

File: utils/src/main/java/org/jboss/windup/util/ZipUtil.java (hunk 1 above)
Comment (carlosthe19916) [reply utils/src/main/java/org/jboss/windup/util/ZipUtil.java]: Changed to private

File: utils/src/main/java/org/jboss/windup/util/ZipUtil.java (hunk 2 above)
Comment (carlosthe19916) [reply utils/src/main/java/org/jboss/windup/util/ZipUtil.java]: Thanks. Fixed and now it says "skip"

**Comment by jmle** [comment]
@carlosthe19916 there are conflicts in this branch - I'd recommend to fix them before it gets stale, and maybe we can merge it

**Review by jmle — APPROVED** [review]

**Pull request merged**